machine_learning/bernoulli_naive_bayes.py

```diff
@@ -0,0 +1,280 @@
+"""
+Naive bayes from scratch on a custom dataset
+
+Naive Bayes Algorithm is a probability based classification technique
+for classifying labelled data. It makes use of Bayes Theorem in order to predict
+the class of the given data. It is called Naive because it makes an assumption that
+the features of the dependent variable are mutually independent of each other.
+Although it is named naive, it is a very efficient model, often used as a baseline
+for text classification and recommender systems.
+
+You can find the detailed notes on this algorithm contributed by me on this notebook: -
+    https://git.io/JTkvO
+"""
+import numpy as np
+import pandas as pd
+
+
+class naive_bayes:
```

Comment:
```suggestion
class NaiveBayes:
```
Python naming...  See CONTRIBUTING.md